client-v2/src/main/java/com/clickhouse/client/api/Client.java

```diff
@@ -208,25 +400,29 @@ private ClickHouseNode getServerNode() {
     }
 
     /**
-     * Ping the server to check if it is alive
+     * Pings the server to check if it is alive
      * @return true if the server is alive, false otherwise
      */
     public boolean ping() {
         return ping(Client.TIMEOUT);
     }
 
     /**
-     * Ping the server to check if it is alive
+     * Pings the server to check if it is alive
      * @param timeout timeout in milliseconds
      * @return true if the server is alive, false otherwise
      */
-    public boolean ping(int timeout) {
+    public boolean ping(long timeout) {
```

Comment:
Should we check the value range?

Reply:
The Math.toIntExact(timeout) will throw exception if value overflows int. 
I would may be add max 10 minutes but is it too restrictive? May be someone will use ping to wakeup an instance - for how long would it wake up?